lib/screens/v2/candidate_generator/elevator/elevator_closures.ex

```diff
@@ -0,0 +1,85 @@
+defmodule Screens.V2.CandidateGenerator.Elevator.ElevatorClosures do
```

Comment:
I don't love the duplicative nature of this name — what do you think of `Elevator.Closures`? If we're aliasing it somewhere outside the `Elevator` namespace and want to make it clear what kind of "closures" these are, we could always alias it `as: ElevatorClosures`.